modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,1073 @@
+= XDCR Conflict Logging
+:description: pass:q[XDCR detects and logs concurrent conflicts that occur in different clusters but on the same document version due to independent modifications by locally connected applications during Active-Active replication. You can manually review and resolve these conflicts to prevent data loss.]
+
+[abstract]
+{description}
+
+During XDCR Active-Active replication, a conflict can occur if different applications connected locally to two different clusters, independently modify the same version of the document at the same time.
+These simultaneous modifications happen within the convergence window*, so the changes made in one document are not reflected in the history of the other.
+
+XDCR detects and logs these mutations, which you can manually review, upsert, and continue with the resolution process.
+
+_*A convergence window is a period during which multiple actors, such as users, applications, or clusters, can modify the same document in an active-active topology, before the changes have fully converged across all clusters._
+
+== Adaptibility of XDCR Conflict Logging
+
+* Conflicts are expected to be rare during XDCR replication. Due to any reason, if there is an upsurge of conflicts, XDCR skips logging conflicts and prioritises replication.
+
+* Conflict logging is asynchronous and attempted on the best-effort basis, which makes it the lowest operational priority.
```

Comment:
I don't think that we should say that conflict logging has the lowest operational priority ... we've already said that it's on a best-effort basis.  Also, I think that it reads better if the order of line 16 and 19 are switched.

* Conflict logging is asynchronous and done on a best-effort basis, meaning, not all conflicts may be logged.

* Conflicts are expected to be rare during XDCR replication. If there is an upsurge of conflicts so that logging conflicts may cause replication issues, XDCR will stop logging conflicts temporarily to prioritize replication.